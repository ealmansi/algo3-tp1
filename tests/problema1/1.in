# D n d1 d2 ... dn
# ... mÃ¡s instancias
# 0

25 7 1 34 2 43 12 54 46
0