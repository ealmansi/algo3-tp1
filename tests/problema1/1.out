# d c
# ... mÃ¡s instancias

34 4